lib/elements/analysis_results_controller.dart

```diff
@@ -118,3 +178,15 @@ class AnalysisResultsController {
     toggle.text = _hideMsg;
   }
 }
+
+class LineInfo {
```

Comment:
You may want to consider a slightly more specific name, like TextRange or SelectionRange or something. Just a thought.